edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/ConnectionManager.cs

```diff
@@ -58,6 +61,9 @@ public async Task AddDeviceConnection(IIdentity identity, IDeviceProxy devicePro
             ConnectedDevice device = this.GetOrCreateConnectedDevice(identity);
             Option<DeviceConnection> currentDeviceConnection = device.AddDeviceConnection(deviceProxy);
             Events.NewDeviceConnection(identity);
+            Metrics.EdgeHubConnectedClientCountIncrement();
```

Comment:
Nit: So the idea of classes like Events/Metrics in each containing classes like ConnectionManager was to reduce the noise in actual code. In that spirit, should we collapse these 3 methods into 1 - Metrics.ClientConnected(id).. which can then call these 3 methods. Same with Metrics.ClientDisconnected(id).

Reply:
Sorry, should have mentioned this earlier.. If you want, you can merge this and I can make a pass and clean up..

Reply:
Sure. My preference is to keep them where they are because its more readable. But feel free to reorg.